src/pvnet_app/data/nwp.py

```diff
@@ -222,3 +215,80 @@ def rename_ecmwf_variables():
         # save back to path
         shutil.rmtree(nwp_ecmwf_path, ignore_errors=True)
         d.to_zarr(nwp_ecmwf_path)
+
+
+def check_model_nwp_inputs_available(
+    data_config_filename: str,
+    t0: pd.Timestamp,
+) -> bool:
+    """Checks whether the model can be run given the available NWP data
+
+    Args:
+        data_config_filename: Path to the data configuration file
+        t0: The init-time of the forecast
+
+    Returns:
+        bool: Whether the NWP timestamps satisfy that specified in the config
+    """
+    input_config = load_yaml_configuration(data_config_filename).input_data
+
+    available = True
+
+    # check satellite if using
+    if hasattr(input_config, "nwp") and (input_config.nwp is not None):
+
+        for nwp_source, nwp_zarr_path in zip(["ukv", "ecmwf"], [nwp_ukv_path, nwp_ecmwf_path]):
+
```

Comment:
Do you mind adding a loggin statement here, perhaps debug, just saying what the next few lines of code are doing